test/export/test_export.py

```diff
@@ -37,23 +37,6 @@ def f(x):
         self.assertTrue(torchdynamo.utils.same(ref, res))
 
     def test_export_constraints_error(self):
-        def invalid_size(x):
-            b = x.item()
-            constrain_as_size(b, min=0, max=5)
-            return torch.full((b, 1), 1)
-
-        inp = (torch.tensor([3]),)
-        with self.assertRaisesRegex(torchdynamo.exc.UserError, "Unable to set min size"):
-            export(invalid_size, inp)
-
-        def invalid_input_conflict_with_inline_constraints(x):
-            b = x.item()
-            constrain_as_size(b, min=2, max=5)
-            return torch.full((b, 1), 1)
-
-        inp = (torch.tensor([6]),)
-        with self.assertRaisesRegex(torchdynamo.exc.UserError, "Invalid value 6 for range"):
-            export(invalid_input_conflict_with_inline_constraints, inp)
```

Comment:
You can still do this test, so long as you actually run the resulting exported module (it should error at runtime)